docker-entrypoint.sh

```diff
@@ -24,6 +24,8 @@ if [ ! -z "${DATABASE_URL}" ]; then
         DATABASE_PATH=$(php -r "echo substr(parse_url(preg_replace('/^([\\w\\\\\\]+)/', 'file', getenv('DATABASE_URL')), PHP_URL_PATH), 1);")
         echo "=====> Path: ${DATABASE_PATH}"
         chmod a+rwx ${DATABASE_PATH}
+        chown www-data:www-data ${DATABASE_PATH} ${DATABASE_DIR}
+        chmod g+w ${DATABASE_DIR}
```

Comment:
Might also work without `DATABASE_DIR` env variable being set manually in `Dockerfile` inferring its value using `dirname`:

```suggestion
        DATABASE_DIR="$(dirname ${DATABASE_PATH})"
        chown www-data:www-data ${DATABASE_PATH} ${DATABASE_DIR}
        chmod g+w ${DATABASE_DIR}
```